### ISSUE TITLE
ladder net g_guass unclear

### ISSUE THREAD

**Issue by thebeancounter** [divamgupta/ladder_network_keras#2]
Can't find any reference to that formula in call and in g_guass in the paper and online, can you explain the origin of the formula 

```
mu = a1 * tf.sigmoid(a2 * u + a3) + a4 * u + a5
v = a6 * tf.sigmoid(a7 * u + a8) + a9 * u + a10

z_est = (z_c - mu) * v + mu
return z_est
```

**Comment by divamgupta** [comment]
Hi, the formula of the denoising function  is given in the appendix of the paper  :  "B Formulation of the Denoising Function"

**Comment by wuhaizhifeng** [comment]
hi, about the cost, you just use the decoder cost(d_cost), however, the c_cost you dont use it? Right?

**Comment by divamgupta** [comment]
We compile the model with categorical cross entropy, hence that loss is
added by keras while compiling.

**Comment by wuhaizhifeng** [comment]
Thanks， I got it.


------------------
 Phd student:   Mu Guoqing
Institute of Advanced Control Technology
School of Control Science & Engineering
Dalian University of Technology (DLUT), P. R. China





 Email： guoqingmu@foxmail.com


 




------------------ Original ------------------
